vars/newrelic.groovy

```diff
@@ -0,0 +1,47 @@
+/***********************************
+
+  newrelic DSL
+
+  notifies newrelic of a deployment
+
+  example usages:
+
+  newrelic(
+    apiKey: 'xxxxxxxxxxxxxxxxxxxx',
+    application: 'myApp',
+    user: 'jenkinsUser'
+  )
+
+  withCredentials([string(credentialsId: 'newrelic-api-key', variable: 'APIKEY')]) {
+    wrap([$class: 'BuildUser']) {
+      newrelic(
+        apiKey: "${APIKEY}",
+        application: 'myApp',
+        user: "${BUILD_USER}"
+      )
+    }
+  }
+
+************************************/
+
+import groovy.json.JsonSlurperClassic
+
+def parseJsonToMap(String json) {
+  final slurper = new JsonSlurperClassic()
+  return new HashMap<>(slurper.parseText(json))
+}
+
+def call(body) {
+  def config = body
+  def shortCommit = sh(returnStdout: true, script: "git log -n 1 --pretty=format:'%h'").trim()
```

Comment:
@jaredbrook we have `shellOut` helper for this, problem here would be that `shortCommit` var has line feed as last character, same applies for line below - see https://github.com/base2Services/ciinabox-pipelines/blob/master/vars/shellOut.groovy